web/src/app/wizard/WizardRouter.js

```diff
@@ -45,8 +45,9 @@ const styles = {
 
 @withWidth()
```

Comment:
Instead of this decorator/higher order component (HOC) adding the width prop to this component, you can use the `useWidth` hook we have created from the util directory:

`import useWidth from '../../util/useWidth'`

and then

`const width = useWidth()`